Makefile

```diff
@@ -1,48 +1,32 @@
 all: test
 
 uninstall:
+	poetry env remove --all
```

Comment:
@benoit-cty As a maintainer, I need to manage virtual envs manually, because I do a lot of cross-library checks for some edge cases hard to reproduce automatically, so I deactivate `poetry`'s env management. That is why I prefer to keep both commands, to make sure all cases are covered.

Reply:
OK, that make sense.

So maybe putting poetry everywhere it's more updating the doc to ask developpers to use `poetry shell` before using `make test` or `make format` ?

Reply:
(BTW, I'm working on a `make` replacement just right after this PR).

Reply:
OK, here is another suggestion : https://github.com/openfisca/extension-template/commit/c6deff5601aae323d1e3402d83fa43c45b7c1221

It keeps no `poetry run` in Makefile for people that do not what Poetry and explain to run `poetry run make lint` for the others.

Reply:
Please take a look at my last push.

Reply:
Nice !

In Makefile you have kept the `poetry run`  is it what you want ?

I'm OK with that but I understand that you do not what it to be able to use `make` without Poetry.

Reply:
I think that, if you use make, you're a contributor. Therefore, you have to use poetry.

However, for whom installs the pack just to add it as a dep, it is expected that you use pip.

That latter case is covered by `tox`.

Reply:
(Mixing `poetry` and `pip` is a bad idea in general. I do it because I'm fighting dragons).